pkg/cluster/internal/loadbalancer/config.go

```diff
@@ -32,26 +32,33 @@ type ConfigData struct {
 
 // DefaultConfigTemplate is the loadbalancer config template
 const DefaultConfigTemplate = `# generated by kind
```

Comment:
this should look familiar, it's probably good enough, but I'm still iterating on it.